gammapy/maps/geom.py

```diff
@@ -400,6 +389,9 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+    def __hash__(self):
+        return hash((self._nodes.tostring(), self._node_type, self._name))
```

Comment:
What is the logic of which properties are hashed here? Why isn't e.g. unit included?
For the nodes array: can't we use the Python object ID, avoiding the `tostring` computation?
I think the assumption is anyways that no-one messes with the nodes array values after init, no?

Reply:
Adressed. As discussed in the dev meeting I changed to using `id()` as a hash.